test/suites/container_devices_disk.sh

```diff
@@ -72,10 +72,10 @@ _container_devices_disk_shift() {
   lxc launch testimage foo-isol1 -c security.idmap.isolated=true
   lxc launch testimage foo-isol2 -c security.idmap.isolated=true
 
-  lxc config device add foo shifted disk pool="${POOL}" source=foo-shift path=/mnt
-  lxc config device add foo-priv shifted disk pool="${POOL}" source=foo-shift path=/mnt
-  lxc config device add foo-isol1 shifted disk pool="${POOL}" source=foo-shift path=/mnt
-  lxc config device add foo-isol2 shifted disk pool="${POOL}" source=foo-shift path=/mnt
```

Comment:
should we keep at least one of the old way to check for compatibility?

Reply:
I'll re-evaluate the tests here, this attempt is pretty weak :slightly_smiling_face: